dask/array/tests/test_fft.py

```diff
@@ -41,6 +41,15 @@ def eq(a, b):
         return c
 
 
+def cmp_dsks(a, b):
+    def key(k):
+        if isinstance(k, str):
+            return (k, -1, -1, -1)
+        else:
+            return k
+    return sorted(a.dask, key=key) == sorted(b.dask, key=key)
```

Comment:
I've been thinking that for a while. Every file implements their own `eq`, and now this function as well.